src/memgraph.cpp

```diff
@@ -624,7 +621,7 @@ int main(int argc, char **argv) {
   auto db_acc = dbms_handler.Get();
 
   memgraph::query::InterpreterContextLifetimeControl interpreter_context_lifetime_control(
-      interp_config, &dbms_handler, repl_state, system,
+      interp_config, settings.get(), &dbms_handler, repl_state, system,
```

Comment:
Does raw ptr vs weak ptr matter here. I've not thought hard enough to figure out if it matters or not.